tortoise/fields/data.py

```diff
@@ -73,19 +75,17 @@ class IntField(Field[int], int):
     """
 
     SQL_TYPE = "INT"
+    VALUE_RANGE = (-2147483648, 2147483647)
```

Comment:
I don't think we should treat `VALUE_RANGE` in the same ways  as `SQL_TYPE`. `SQL_TYPE` is present for all fields but `VALUE_RANGE` just for integers. Having it as a `VALUE_RANGE` as a class field might give an impression that this is a part of the interface of all field classes but it isn't. It also makes it possible to get the constraints in two ways: through `VALUE_RANGE` and through `constraints` making it harder to know which is the right one to use. I would suggest to revert this change and keep constraints in the constraints method.